app/serializers/youth_risk_assessment_serializer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class YouthRiskAssessmentSerializer < FrameworkAssessmentSerializer
+  belongs_to :profile, serializer: V2::ProfileSerializer
```

Comment:
this was required due to autoloading the mirror relationship in v2 serializers, which wasn't working as expected. When we move off v2 this can go back to the parent class

Reply:
I'll a comment so to explain and a so we don't forget to do when we move off 👍